src/components/alerts/Notifications.vue

```diff
@@ -165,7 +165,7 @@
                          if (this.checkIfExistsAlert("web-assembly-warning")) break;
 
                          this.alerts.push("web-assembly-warning");
-                         Notification.addAlert("web-assembly-warning", "warn", "Performance Warning", "<b>WebAssembly is not supported</b>. You use an old browser or one that doesn't support WebAssembly. Install Chrome/Firefox/Safari and mining increases with 70%", 86400000);
+                         Notification.addAlert("web-assembly-warning", "warn", "Performance Warning", "<b>WebAssembly is not supported</b>. You use an old browser or one that doesn't support WebAssembly. Install Chrome/Firefox/Safari/Opera and mining will increse with ~70%", 86400000);
```

Comment:
Notification.addAlert("web-assembly-warning", "warn", "Performance Warning", "<b>WebAssembly is not supported</b>. You use an old browser or one that doesn't support WebAssembly. Install Chrome/Firefox/Safari/Opera and mining will increase by ~70%", 86400000);